spec/end_to_end/other_income_spec.rb

```diff
@@ -105,21 +105,35 @@ def submit_data_to_cfe
     end
   end
 
-  # Note - this is an incomplete test, it is just a 'proof of concept' for the end2end tests
-  # in terms of the structure and how the existing end2end tests can re-use 98% of the code
+  # CFE gives back montlhy figures and ignores frequency (foo[your input value if frequency is weekly] x 52[weeks] / 12[months] = bar[CFE output])
```

Comment:
"montlhy" is a typo. Also, it doesn't ignore frequencies, on the contrary it converts the frequency given to a monthly equivalent.